sysconfig/cri-resource-manager

```diff
@@ -0,0 +1,8 @@
+# Name of the policy to activate.
+POLICY=topology-aware
+# Any extra options for the active policy.
+POLICY_OPTIONS="--reserved-resources=cpu=750m"
+# Debug flags to enable.
+DEBUG_OPTIONS="--logger-debug resource-manager,policy,cache,$POLICY"
+# Config file for accessing the cluster through the API server.
+KUBE_CONFIG="--kubeconfig /root/.kube/config"
```

Comment:
cri-rm should not be directly utilizing kubeconfig.

Reply:
Yes. It is not used by cri-resmgr. That KUBE_CONFIG is only used by cri-resmgr-agent.